src/main/java/org/broadinstitute/hellbender/tools/spark/sv/FindBreakpointEvidenceSpark.java

```diff
@@ -327,11 +339,14 @@ private void addAssemblyQNames(
 
     /** write a FASTQ file for an assembly */
     private static void writeFastq( final Tuple2<Integer, List<byte[]>> intervalAndFastqs,
-                                    final String outputDir ) {
+                                    final String outputDir, final int maxFastqSize ) {
         final List<byte[]> fastqsList = intervalAndFastqs._2;
         SVFastqUtils.sortFastqRecords(fastqsList);
 
-        final String fileName = outputDir + "/assembly" + intervalAndFastqs._1 + ".fastq";
+        String fileName = outputDir + "/assembly" + intervalAndFastqs._1 + ".fastq";
+        final int fastqSize = fastqsList.stream().mapToInt(fastqRec -> fastqRec.length).sum();
+        if ( fastqSize > maxFastqSize ) fileName += ".toobig";
```

Comment:
If I'm getting this right we're still writing out the big FASTQ file but with a different extension? I think that will still cause a problem for the downstream tool since it tries to load all files in the given directory. Maybe put this in a different directory, or just skip writing it altogether? Sounds like this might be moot with your recent changes anyway.